baseplate/context/cassandra.py

```diff
@@ -148,8 +153,25 @@ def execute_async(self, query, parameters=None, timeout=_NOT_SET):
         future.add_errback(_on_execute_failed, span)
         return future
 
-    def prepare(self, query):
+    def prepare(self, query, cache=True):
+        """Prepares a CQL statement.
+
+        :param bool cache: If set to True (default), prepared statements will be
```

Comment:
when do you see this being set to false? if there are known specific situations it may be good to pop that into the docs.